content/installation/upgrade.md

```diff
@@ -11,6 +11,13 @@ Custom installations do not support client-side upgrade functionality.
 Use [`kubectl directpv migrate`]({{< relref "/command-line/migrate.md" >}}) to migrate the old resources to a new installation.
 {{< /admonition >}}
 
+{{< admonition title="Pod Security Policies" type="note" >}}
+Kubernetes deprecated the `PodSecurityPolicy` feature in v1.21, then removed it entirely in v1.25.
+The Kubernetes documentation [recommends]](https://kubernetes.io/docs/concepts/security/pod-security-policy/) [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/) as a replacement.
```

Comment:
Is this maybe a mistake to have a double URL here? `[recommends]]` seems like maybe you intended to link something else.

Either way I'd restructure to separate the two URLs in the sentence.

Reply:
Had already fixed the errant double `]]`. But reworked the sentence to separate the links as suggested.